auth/client/src/main/java/org/wildfly/security/auth/client/AuthenticationConfiguration.java

```diff
@@ -1147,10 +1147,7 @@ public AuthenticationConfiguration useSaslProtocol(String saslProtocol) {
     }
 
     public AuthenticationConfiguration useWebServices(Map<String, ?> webservicesProperties) {
-        final HashMap<String, Object> newMap = new HashMap<>();
-        newMap.putAll(webservicesProperties);
-        newMap.values().removeIf(Objects::isNull);
-        return new AuthenticationConfiguration(this, SET_WEBSERVICES_PROPS, optimizeMap(newMap));
+        return useWebServicesProperties(webServicesProperties);
```

Comment:
@nidhiazad I am very sorry, this issue should be closed as "won't do", this is not thread safe operation. Please close the jira issue as won't do, thank you!

Reply:
okay. Thanks for the update.

Reply:
@nidhiazad Thank you for closing the jira issue. You can remove this commit from this PR. Other 2 issues seem okay to me, only you need to rename the commit that fixes the link to jira issues. Let us know if you need any help, thank you!